presto-native-execution/presto_cpp/main/SessionProperties.cpp

```diff
@@ -299,11 +299,32 @@ SessionProperties::SessionProperties() {
       kQueryTraceTaskRegExp,
       "The regexp of traced task id. We only enable trace on a task if its id"
       " matches.",
-      BIGINT(),
+      VARCHAR(),
```

Comment:
was that an existing bug that we are fixing along with introducing new session properties?

Reply:
Yeah that was a bug..

Reply:
Does it make sense to add some release note around the bug fix? Or its minor enough that no release note needed?